frontend/src/App.js

```diff
@@ -14,16 +12,17 @@ function App() {
   return (
     <div>
       <Auth0Provider
-        domain="dev-kce2a8b1cexzb43y.us.auth0.com"
         clientId="LY950lpDF68jgnJEGIWUtUaMunlpTeFs"
```

Comment:
Add `domain` & `clientId` to `.env` and use `process.env.` here, it's sensitive details and should not be available on GitHub.